cli/server_request_command.go

```diff
@@ -113,6 +117,10 @@ func configureServerRequestCommand(srv *fisk.CmdClause) {
 	healthz := req.Command("jetstream-health", "Request JetStream health status").Alias("healthz").Action(c.healthz)
 	healthz.Flag("js-enabled", "Checks that JetStream should be enabled on all servers").Short('J').BoolVar(&c.jsEnabled)
 	healthz.Flag("server-only", "Restricts the health check to the JetStream server only, do not check streams and consumers").Short('S').BoolVar(&c.jsServerOnly)
+
+	profilez := req.Command("profile", "Run a profile").Action(c.profilez)
+	profilez.Flag("profile", "Specify the name of the profile to run").StringVar(&c.profileName)
```

Comment:
since its required and has no default we could make this an `Arg()`  and then have `EnumVar(&c.profileName, "cpu", "mem", ....)` so we'll get validtion etc.  In the help portiion here I tend to list the valid enum values in the help string like `"The profile name to request (mem, cpu, ....)"`

Reply:
Switching to `Arg()` definitely makes sense. I did wonder about validating the profile name clientside but then if we change profiles in the server later, that would also require a NATS CLI update. Passing through the name and letting the server decide if it's acceptable probably makes more sense here.

Reply:
I tend to favor user friendlyness here over the work to keep it friendly :) But yeah this is niche enough and these `nats server req` stuff are advanced enough usage that I think its ok without the enum.

Reply:
But we should show what is valid in --help at least, even if we dont validate so that as we add more peole arent stuck that they cant access new ones if we tell them in support - and then presumably support will result in a fix issue here